ui/src/engine/pearl/mod.rs

```diff
@@ -223,6 +223,34 @@ impl EventHandler for Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
             Event::Resume => {
                 self.paused = false;
             }
+            Event::Beacon => {
+                let master_volume = self.sound.volume();
+                self.sound.set_master_volume(100);
```

Comment:
if i'm not mistaken, 100 is too loud on Pearl and we risk blowing the fuse

the value is clamped on the backend side on normal usage

would `50` work for you?